policies/source_image_task.rego

```diff
@@ -0,0 +1,18 @@
+package policies.source_image_task
+
+import data.lib
+
+# METADATA
+# title: Verify the source-image-verify task accepts an image
+# description: |-
+#    Verify the source-image-verify task accepts a particular image
+#    as an input param.
+# custom:
+#   short_name: disallowed_input_image
+#   failure_msg: Task '%s' does not contain '%s' as a param
+warn[result] {
+	image_sha = data.source_image_verify.input_image
+	task_name = data.source_image_verify.task_name
```

Comment:
I don't understand where data.source_image_verify comes from here.

Reply:
What's a typical value for input_image and task_name?

Reply:
The `input_image` is coming from the release pipeline, so my initial thought was the release pipeline had to pass this information down. Zoran brought up pulling it from the attestation instead. I need to test it out, but it looks like that's doable and a better solution. So, in short, this code most likely is changing.

Reply:
Ack, ok.

Reply:
If it is a manually set value, maybe it can live in the EnterpriseContractPolicy .